modules/Bio/EnsEMBL/Hive/Queen.pm

```diff
@@ -222,6 +222,7 @@ sub create_new_worker {
     }
 
     $worker->set_log_directory_name($hive_log_dir, $worker_log_dir);
+    $worker->set_temp_directory_name( $worker_base_temp_dir || $meadow->config_get('BaseTempDirectory') );
```

Comment:
If BaseTempDirectory isn't set in the config, and -worker_base_temp_dir isn't set, how does temp_directory_name get populated in the worker table?

Reply:
`set_temp_directory_name` will fallback to the system's default. See https://github.com/Ensembl/ensembl-hive/blob/experimental/worker_temp_dir/modules/Bio/EnsEMBL/Hive/Worker.pm#L937-L940